lib/iris/coords.py

```diff
@@ -1697,15 +1697,25 @@ def from_coord(coord):
         return new_coord
 
     def _sanitise_array(self, src, ndmin):
-        # Ensure the array is writeable.
-        # NB. Returns the *same object* if src is already writeable.
-        result = np.require(src, requirements='W')
-        # Ensure the array has enough dimensions.
-        # NB. Returns the *same object* if result.ndim >= ndmin
-        result = np.array(result, ndmin=ndmin, copy=False)
-        # We don't need to copy the data, but we do need to have our
-        # own view so we can control the shape, etc.
-        result = result.view()
+        if is_lazy_data(src):
```

Comment:
At present, lazy bounds aren't achievable, so I can't get the intended tests working without changing ***something***.